javalib/src/main/scala/java/util/concurrent/ConcurrentSkipListSet.scala

```diff
@@ -0,0 +1,423 @@
+package java.util.concurrent
+
+import java.util._
+import java.lang.Comparable
+import scala.collection.mutable
+import scala.math.Ordering
+
+import scala.collection.JavaConversions._
+
+class ConcurrentSkipListSet[E] protected (ordering: Ordering[E], provided: Option[Comparator[E]])
```

Comment:
Is there a reason why `provided` is an `Option` instead of just using `null` when it's not provided? Also maybe it should be called `_comparator`?